vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -434,6 +451,17 @@ func (e *StandardTxExecutor) AddPermissionlessValidatorTx(tx *txs.AddPermissionl
 	avax.Consume(e.State, tx.Ins)
 	avax.Produce(e.State, txID, tx.Outs)
 
+	if e.Config.LiteSyncPrimaryNetwork &&
+		tx.Subnet == constants.PrimaryNetworkID &&
+		tx.Validator.NodeID == e.Ctx.NodeID {
+		e.Ctx.Log.Warn("verified transaction that would shutdown this node",
```

Comment:
Is shutdown really the best behavior if anyone can make a NodeID a primary network validator?

Reply:
I'd prefer we only warn and just not respond with chits.

Reply:
sgtm